### ISSUE TITLE
Missing NEI/JEI Support

### ISSUE THREAD

**Issue by Delocuro** [MinestrapTeam/Minestrappolation-4#105]
The following blocks should have NEI support but currently don't.
- [ ] Crusher (Was added but kept crashing, so we'll need to figure out how to fix that)
- [ ] Splitter
- [ ] Frost Generator (Will need its own NEI interface somehow... maybe just two slots, one with the block and one with the freezing result, with an arrow in between the two?)
- [ ] Sifter and Boulder outcomes would be useful, but like the Frost Generator we'd need to figure out some form of GUI to represent them.
- [ ] Dimensional Fishery items would be really useful to have too, but those'd also need a GUI of some sort.

**Comment by Delocuro** [comment]
Removing this from the 4.3.0 milestone seeing as NEI hasn't updated to 1.8.8 yet.

**Comment by medsal15** [comment]
Maybe do JEI instead?

**Comment by Delocuro** [comment]
That's the current plan -- we just have to figure out how to set it up since their code is slightly different.

**Comment by bluebear94** [comment]
The Sifter interface could be like IC2's scrapbox, showing the chances of getting the item in question. The Boulder interface could be similar, but show the appropriate boulder.

**Comment by Delocuro** [comment]
Yeah, that's probably how we'll end up setting those up. Same goes for the Dimensional Fishery, though that one could get complicated given the sheer quantity of potential drops.

**Comment by MaegnusTheBlackMage** [comment]
what is still needed to make this happen? With a lot of vanilla recipes removed and no JEI/NEI support to show the new recipes it is really hard to play this in a server environment without turning the vanilla recipes back on. Im interested in helping if im able

**Comment by werty1124** [comment]
For Minestrappolation 4.3 adding the NEI/JEI plugins to actually support our new blocks

**Comment by MaegnusTheBlackMage** [comment]
Mentioned the option of adding support for this mod to JEIAddons though i may try to work on it if they dont. #https://github.com/mezz/JEIAddons/issues/16
